### PR TITLE
(Docs)Improve home section

### PR DESCRIPTION
**Pull request by TIMMLOPK**
### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation Only
- [ ] Refactor

### CODE REVIEW

File: apps/docs/src/components/home/built-in-stitches.tsx [hunk 1]
```diff
@@ -3,17 +3,15 @@ import NextLink from 'next/link';
...
-  Spacer
-} from '@nextui-org/react';
+import * as react from '@nextui-org/react';
 import landingContent from '@content/landing';
-import { CodeDemo, Blockholder } from '@components';
+import { Blockholder } from '@components';
+
+import dynamic from 'next/dynamic';
+
+const DynaminCodeDemo = dynamic(() => import('../code-demo/code-demo'), {
```
Comment (jrgarciadev) [inline apps/docs/src/components/home/built-in-stitches.tsx:12]: Hey @TIMMLOPK thanks for contributing to NextUI

Is not necessary to dynamically import the `code-demo` component due to we are already showing it just when is visible in the viewport line:49

File: apps/docs/src/components/home/built-in-stitches.tsx [hunk 2]
```diff
@@ -3,17 +3,15 @@ import NextLink from 'next/link';
...
-import {
-  Grid,
-  Row,
-  Col,
-  Link,
-  Text,
-  Collapse,
-  Spacer
-} from '@nextui-org/react';
+import * as react from '@nextui-org/react';
```
Comment (jrgarciadev) [inline apps/docs/src/components/home/built-in-stitches.tsx:6]: Please leave the separated imports

**Comment by jrgarciadev** [comment]
Hey @TIMMLOPK I appreciate your contribution, however, is not necessary to dynamically import the `code-demo` component due to we are already showing it just when is visible in the viewport `line:49`, could you explain why did you remove the `accordion` in collapse group?

**Comment by TIMMLOPK** [comment]
> Hey @TIMMLOPK I appreciate your contribution, however, is not necessary to dynamically import the `code-demo` component due to we are already showing it just when is visible in the viewport `line:49`?

Hey,I had noticed [home](https://github.com/nextui-org/nextui/tree/main/apps/docs/src/components/home)custom section still use dynamic import code demo

**Comment by jrgarciadev** [comment]
Hey, @TIMMLOPK looks like I forgot to remove it only in `custom-themes.tsx` https://github.com/nextui-org/nextui/blob/main/apps/docs/src/components/home/custom-themes.tsx, would you like to send a new PR removing it? 🙌🏻

**Comment by TIMMLOPK** [comment]
> Hey, @TIMMLOPK looks like I forgot to remove it only in `custom-themes.tsx` https://github.com/nextui-org/nextui/blob/main/apps/docs/src/components/home/custom-themes.tsx, would you like to send a new PR removing it? 🙌🏻

I am pleased to open new PR

**Pull request closed without merging**